### PR TITLE
Update to PHPStan level 8.

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

File: src/DomainModel/DomainModelAwareTrait.php [hunk 1]
```diff
@@ -54,10 +54,10 @@ public function loadService($model, array $constructorArgs = [], $assignProperty
             return $domainModel;
         }
 
-        list(, $name) = pluginSplit($model);
+        [, $name] = pluginSplit($model);
 
         if (isset($this->{$name})) {
-            trigger_error(__CLASS__ . '::$%s is already in use.', E_USER_WARNING);
+            trigger_error(self::class . '::$%s is already in use.', E_USER_WARNING);
```
Comment (saeideng) [inline src/DomainModel/DomainModelAwareTrait.php]:
```suggestion
            trigger_error(static::class . '::$%s is already in use.', E_USER_WARNING);
```

why not the  `static` ??
it refers to last class, not where this trait used

Comment (dereuromark) [reply src/DomainModel/DomainModelAwareTrait.php]: This was done by sniffers.

Comment (saeideng) [reply src/DomainModel/DomainModelAwareTrait.php]: also forgot to use the `sprintf()`  here

Comment (saeideng) [reply src/DomainModel/DomainModelAwareTrait.php]: I checked again replace of `__CLASS__` exactly  is `self::class`, but probably `static::class` is true here, not tested locally

**Pull request merged**